recipes/ckzg/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ckzg" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ckzg-{{ version }}.tar.gz
+  sha256: 4295acc380f8d42ebea4a4a0a68c424a322bb335a33bad05c72ead8cbb28d118
+  patches:
+    - setup.py.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('clang') }}
+    - {{ stdlib('c') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - libblst
+    - libblst-headers
```

Comment:
Suggesting dropping these per the discussion in this thread: https://github.com/conda-forge/staged-recipes/pull/26068#discussion_r1810520733

```suggestion
```